common/dtd/ldmlSupplemental.dtd

```diff
@@ -17,7 +17,7 @@ Except as contained in this notice, the name of a copyright holder shall not be
 <!ATTLIST version number CDATA #REQUIRED >
 	<!--@MATCH:any-->
     <!--@METADATA-->
-<!ATTLIST version cldrVersion CDATA #FIXED "36" >
+<!ATTLIST version cldrVersion CDATA #FIXED "37" >
 	<!--@MATCH:version-->
     <!--@VALUE-->
 <!ATTLIST version unicodeVersion CDATA #FIXED "12.1.0" >
```

Comment:
unicodeVersion needs to be 13.0 for CLDR 37

Reply:
Fixed...